src/condor_tools/htcondor_cli/dagman.py

```diff
@@ -64,19 +104,44 @@ class Status(Verb):
         },
     }
 
+    def multi(self, num, past=False):
+        if num > 1 and not past:
+            return f"{num:3.0f} are"
+        elif num > 1 and past:
+            return f"{num:3.0f} have"
+        elif not past:
+            return f"{num:3.0f} is"
+        else:
+            return f"{num:3.0f} has"
```

Comment:
I approve of this attention to good grammar.

Reply:
Thanks :)